script/eslint-plugin-va/rules/use-resolved-path.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable va/use-resolved-path */
+// self rule is disabled because it contains the string needed for comparison
+const MESSAGE = 'Use resolved path and remove unnecessary parent path';
+
+function isIncluded(val, path) {
+  const isString = str => typeof str === 'string';
+  if (!isString(val) || path === null) {
+    return false;
+  }
+  return val.includes(path);
+}
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Use resolved path and remove unnecessary parent path',
+      category: 'best practices',
+      recommended: true,
+    },
+    schema: [
+      {
+        properties: {
+          applicationsPath: {
+            description: 'report path containng `applications`',
+            type: 'boolean',
+          },
+          platformPath: {
+            description: 'report path containng `platform`',
+            type: 'boolean',
+          },
+          sitePath: {
+            description: 'report path containng `site`',
+            type: 'boolean',
+          },
+        },
+      },
+    ],
+  },
+  create(context) {
+    const _ref = context.options[0] || {};
+    const applicationsPath = _ref.applicationsPath ? '../applications/' : null;
```

Comment:
Can we use a config for this instead of hardcoding things. Where are these aliases configured? I only see `vet360` configured in the [`.babelrc`](https://github.com/department-of-veterans-affairs/vets-website/blob/7ff9874b2da7eef36997a9033ee96c043c5a915f/.babelrc#L40)

Reply:
Will do!